### PR TITLE
Update cluster example manifest to align with current usage

### PR DESCRIPTION
**Pull request by rtheis**

### CODE REVIEW

**Review by jonesbr17 — APPROVED** [review]

File: cluster.yaml.example [hunk 1]
```diff
@@ -179,29 +168,26 @@ kmsLivenessProbe:
   timeoutSeconds: 160
   failureThreshold: 3
   successThreshold: 1
+controllerManagerLivenessProbe:
+  initialDelaySeconds: 300
+  timeoutSeconds: 160
 portierisLivenessProbe:
```
Comment (jonesbr17) [inline cluster.yaml.example:174]: I don't see a `portierisLivenessProbe` in https://github.ibm.com/alchemy-containers/armada-openshift-master/blob/master/roles/hosted-master/templates/cluster.yaml.j2

Comment (rtheis) [reply cluster.yaml.example:174]: Good catch.  I'll follow-up with Bill.

Comment (rtheis) [reply cluster.yaml.example:174]: https://github.ibm.com/alchemy-containers/armada-deploy/issues/7648

**Pull request merged**